agri-store-react/src/Components/Authantication.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { Navigate, useLocation } from "react-router-dom";
+
+function Authantication({ children }) {
```

Comment:
There is typo error `Authantication` . It should be `Authentication`

Reply:
sure will update in next PR